docker/Dockerfile.linux.amd64

```diff
@@ -0,0 +1,19 @@
+FROM amd64/alpine:edge
+
+RUN apk update && \
+	apk upgrade && \
+	apk add ca-certificates mailcap && \
+	rm -rf /var/cache/apk/* && \
+	echo 'hosts: files dns' >| /etc/nsswitch.conf
+
+LABEL maintainer="ownCloud GmbH <devops@owncloud.com>" \
+  org.label-schema.name="oCIS Accounts" \
+  org.label-schema.vendor="ownCloud GmbH" \
+  org.label-schema.schema-version="1.0"
+
+EXPOSE 9120 9124
```

Comment:
No port matching for https://github.com/owncloud/ocis/wiki/Ports?

Reply:
Need to be reserved indeed. Was focused on having something rolling 🎢

Reply:
done 👍